sdk/synapse/test-resources.json

```diff
@@ -206,7 +205,7 @@
             },
             "properties": {
                 "defaultDataLakeStorage": {
-                    "accountUrl": "[variables('defaultDataLakeStorageAccountUrl')]",
+                    "accountUrl": "[reference(resourceId('Microsoft.Storage/storageAccounts', variables('uniqueDataLakeStorageAccountName')), '2019-06-01').primaryEndpoints.dfs]",
```

Comment:
Tip: you don't need to use `resourceId` and all it requires if you're adding the resource in this template. See sdk/keyvault/test-resources.json for several instances where I used the abbreviated form. This isn't wrong - just long. :)